docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +77,64 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+RabbitMQ Installation
+.....................
+
+If in ``verdi status`` RabbitMQ is not connected, first check that RabbitMQ is actually running:
+
+.. code-block:: console
+
+   $ rabbitmqctl status
```

Comment:
If RabbitMQ is installed globally by package manager, `root` is needed to run this command, or `sudo rabbitmqctl status`.